lib/skywalking/util.lua

```diff
@@ -80,30 +80,21 @@ local random_seed = function ()
     return seed
 end
 
-math.randomseed(random_seed())
-
-function _M.newID()
-    return {timestamp(), math.random(0, MAX_ID_PART2), math.random(0, MAX_ID_PART3)}
+local newID = function()
+    return timestamp() .. '.' .. math.random(0, MAX_ID_PART2) .. '.' .. math.random(0, MAX_ID_PART3)
 end
 
--- Format a trace/segment id into an array.
--- An official ID should have three parts separated by '.' and each part of it is a number
-function _M.formatID(str)
-    local regex = '.'
-    if _M.is_ngx_lua then
-        regex = [[\.]]
-    end
-    local parts = split(str, regex)
-    if #parts ~= 3 then
-        return nil
-    end
+math.randomseed(random_seed())
 
-    return parts
+-- for Nginx Lua
+local ok, uuid = pcall(require, "dependencies/lua-resty-jit-uuid")
+if ok then
+    uuid.seed()
+    newID = function()
+        return uuid.generate_v4()
+    end
```

Comment:
```
if not ok then
    return timestamp() .. '.' .. math.random(0, MAX_ID_PART2) .. '.' .. math.random(0, MAX_ID_PART3)
end
```
is better.

Reply:
When does this go to wrong side?

Reply:
When the unit test, or the nginx not running on the LuaJIT environment.